passport/Google.js

```diff
@@ -0,0 +1,26 @@
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+
+import User from '../models/User';
+import config from '../config/keys';
+
+export default new GoogleStrategy({
+  clientID: config.google.clientID,
+  clientSecret: config.google.clientSecret,
+  callbackURL: '/auth/google/success',
+}, (accessToken, refresh, profile, done) => User.findOne({ googleId: profile.id })
```

Comment:
`findOrCreate` can do the job here since it's probably internally optimized.

Reply:
I agree with using `findOrCreate`, though I don't know if mongoose has findOrCreate